python/dgl/contrib/cugraph/nn/conv/entity_sample.py

```diff
@@ -0,0 +1,161 @@
+import argparse
+from time import time
+import torch as th
+import torch.nn as nn
+import torch.nn.functional as F
+from torchmetrics.functional import accuracy
+
+import dgl
+from dgl.data.rdf import AIFBDataset, MUTAGDataset, BGSDataset, AMDataset
+from dgl.dataloading import MultiLayerNeighborSampler, DataLoader
+from dgl.contrib.cugraph.nn import RelGraphConvOps
```

Comment:
I think we should make it a point to show that only the import for `dgl.contrib.cugraph.nn`  changes and nothing else to make it clear to the users.

Reply:
[6a95973](https://github.com/rapidsai/dgl/pull/15/commits/6a95973a122ee769096329df2c35ef258d31fe38) fixed the imports to match the DGL example.